### ISSUE TITLE
Adding the mod Flat Colored Blocks for building purposes. 

### ISSUE THREAD

**Issue by GodThisTakesTime** [Darkosto/SkyFactory4Issues#908]
I would like for the mod Flat Colored Blocks to be added for the ease of building with chisel and bits. As it would not interrupt the flow of the game at all i think it could be added without negatively affect anyone.

Mod Details
Name: Flat Colored Blocks
CurseForge Link: https://minecraft.curseforge.com/projects/flat-colored-blocks/files/2483466

**Comment by btbenedi** [comment]
Yes please, went looking for them earlier today and was sad to see them not there

**Comment by Mementh** [comment]
more decoration is good
